artifactory/commands/terraform/terraformpublish.go

```diff
@@ -0,0 +1,304 @@
+package terraform
+
+import (
+	"github.com/jfrog/gofrog/parallel"
+	commandsutils "github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/utils"
+	commandutils "github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	clientservicesutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	specutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+type TerraformPublishCommandArgs struct {
+	namespace  string
+	moduleName string
+	provider   string
+	tag        string
+	exclusions []string
+}
+
+type TerraformPublishCommand struct {
+	TerraformCommand
+	*TerraformPublishCommandArgs
+	commandName string
```

Comment:
There's only one command name, so why do we need a variable for storing it?
This would have made sense if we multiple commands that used this struct.

Reply:
Don't we need a command name for the "usage report"?